apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -47,17 +47,39 @@ function getNavBar(siteSettings, variant) {
   return {
     logo: imageFromMedia(title, primaryLogo.url),
     drawerLogo: imageFromMedia(title, drawerLogo.url),
+    explorePageUrl: slug,
```

Comment:
1. Isn't the selected explore page in `siteSettings`?
2. I don't think `explorePageUrl` is a correct name. It's not a URL; may be just a path or path segment. And explore is a UI name, I think as far as backend/HURUmap is concerned this is related to profile(s), no?